scripts/generate-policy-config.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import oyaml as yaml
+import shutil
+import os
+import sys
+
+#an array of directories you want to generate policies for. Please make sure ONLY the directories you want exist here. if there's entries here, delete it and add yours.
+directory = ['./deploy/rbac-permissions-operator-config',
+        './deploy/osd-cluster-admin'
+        ]
```

Comment:
Will be a little easier to read in my opinion.

```suggestion
directory = [
    './deploy/rbac-permissions-operator-config',
    './deploy/osd-cluster-admin'
]
```